site/en/docs/crux/dashboard/index.md

```diff
@@ -0,0 +1,152 @@
+---
+# Required
+layout: 'layouts/doc-post.njk'
+
+# Required
+title: CrUX Dashboard
+
+# Required
+# This appears in the ToC of the project landing page at
+# /docs/[project-name]/. It also appears in the <meta description> used in
+# Google Search.
+description: >
+  Using the CrUX Dashboard for visualizing BigQuery CrUX data
+
+# Optional
+# This appears below the title and is an optional teaser
+subhead: >
+  Using the CrUX Dashboard for visualizing BigQuery CrUX data
+
+# Required
+date: 2022-11-10
+
+# Optional
+# Include an updated date when you update your post
+#updated: 2022-11-09
+
+# Optional
+# How to add a new author
+# https://developer.chrome.com/docs/handbook/how-to/add-an-author/
+# authors:
+
+# Optional
+# How to a new tag
+# https://developer.chrome.com/docs/handbook/how-to/add-a-tag/
+tags:
+  - web-vitals
+  - crux
+---
+
+The CrUX Dashboard is a [Looker Studio](https://cloud.google.com/looker-studio) (formerly Data Studio) dashboard that links to the raw origin-level CrUX data on [BigQuery](/docs/crux/bigquery/) and the visualizes the data for you. It eliminates the need for users of the dashboard to write any queries or generate any charts. Everything is built for you; all you need is to provide an origin and the dashboard will be generated for you.
```

Comment:
Think we need the "(formerly Data Studio)"? Think people will know it as DataStudio for some time but maybe should go all in on Looker rather than living in the past...

Reply:
It's a new change so I like the "formerly Data Studio" part.